plugins/filter/vault_login_token.yml

```
@@ -0,0 +1,77 @@
+# (c) 2022, Brian Scholer (@briantist)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+---
+DOCUMENTATION:
+  name: vault_login_token
+  short_description: Extracts the Vault token from a login or token creation
+  version_added: 2.2.0
+  description:
+    - Extracts the token value from the structure returned by a Vault token creation operation.
+  seealso:
+    - module: community.hashi_vault.vault_login
+    - module: community.hashi_vault.vault_token_create
+    - ref: community.hashi_vault.vault_login lookup <ansible_collections.community.hashi_vault.vault_login_lookup>
```

For plugins **in filter and test docs**, you can use this syntax:
```suggestion
    - plugin: community.hashi_vault.vault_login
      plugin_type: lookup
```
If you use it in other plugin types or modules, the sanity tests will scream at you (but the docs build won't mind).

---

Oh nice! That is pretty annoying that it can't be used for other plugin types... I use the original syntax to link to those pretty commonly and it'd be great to change them to a semantic alternative.

---

The problem with changing the sanity test to allow it for other plugin types is that it will require changes to Automation Hub and potential other docs consumers, and similar as for the semantic markup changes, nobody seems to be interested in doing them (and I still have not the slightest clue which part of the AH codebase is responsible for rendering the docs...).

---

Well.. that's disappointing.

Anyway, I've updated the lookup references in this file at least.